plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d

```diff
@@ -1522,8 +1510,26 @@ auto spawnOverviewActor(OverviewActor.Impl self, FlowControlActor.Address flowCt
         import dextool.plugin.mutate.backend.database.type : FileScore;
 
         auto fileScores = ctx.state.get.db.getMutationFileScoreHistory();
+
         ctx.state.get.files.toIndex(content, HtmlStyle.fileDir, fileScores);
 
+        //Change column: timeframe and value calculation handling
+        auto mut_data = appender!(string[])();
+        mut_data.put("var file_score_data = {};");
+
+        if (fileScores.length > 0) {
```

Comment:
because the foreach loop is only executed if there are any values in fileScores this check isn't needed